man/kernel-install.xml

```diff
@@ -497,6 +497,25 @@
           <xi:include href="version-info.xml" xpointer="v198"/>
           </listitem>
       </varlistentry>
+      <varlistentry>
+        <term>
+          <filename>/usr/lib/kernel/devicetree</filename>
+          <filename>/etc/kernel/devicetree</filename>
+        </term>
+          <listitem>
+            <para>Read by <filename>90-loaderentry.install</filename>. The content of the file
+            <filename>/etc/kernel/devicetree</filename> specifies the device tree blob to install with the
+            kernel and use at boot. The name of the device tree blob is specified relative to where the
+            kernel package installs them and are taken from the same package as the kernel being
+            installed. Eg. <literal>broadcom/bcm2711-rpi-4-b.dtb</literal> installs the device tree
+            blob for the Raspberry Pi 4 Model B. If <filename>/etc/kernel/devicetree</filename> does not
+            exist, <filename>/usr/lib/kernel/cmdline</filename> is used. If that also does not exist, no
+            device tree blob is installed. <varname>$KERNEL_INSTALL_CONF_ROOT</varname> may be used to
+            override the path.</para>
```

Comment:
override which path?

Reply:
This is changed in a later patch along with all the other places that mention $KERNEL_INSTALL_CONF_ROOT.